spec/spec_helper.rb

```diff
@@ -5,27 +5,28 @@
 require 'dotenv'
 require 'simplecov'
 require 'securerandom'
+require 'aws-sdk-sqs'
```

Comment:
Curious how was this working before, given tests rely on SQS 🤔

Reply:
Not sure, but without it the pipeline was failing

Reply:
I do [see](https://github.com/WeTransfer/sqewer/blob/master/lib/sqewer/connection.rb#L47) the gem being required when returning a client, so I find it strange that now all of a sudden tests fail without this extra/duplicate `require` 🤔

Reply:
It's inside a method which makes tests order important, but it shouldn't be the case

Reply:
Okay. Let's tackle this separately. Thanks for the explanations 👍